src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -670,19 +827,27 @@ def __call__(
         lora_scale = (
             self.joint_attention_kwargs.get("scale", None) if self.joint_attention_kwargs is not None else None
         )
+        do_true_cfg = true_cfg > 1 or negative_prompt is not None
```

Comment:
```suggestion
        do_true_cfg = true_cfg > 1
```

don't think we should apply it even `true_cfg` <=1 but user passed a `negative_prompt`

Reply:
It was `true_cfg > 1 and negative_prompt is not None` but v2 of the model seems to give bad results when true cfg > 1 but it still needs to use true cfg to get good results